lib/l10n/intl_it.arb

```
@@ -838,5 +869,57 @@
+  "@failedSendMessage": {
+    "placeholders": {
+      "error": { "type": "String" }
+    }
+  },
+  "waitingAdminAssignmentInput": "In attesa dell'assegnazione dell'amministratore...",
+  "typeYourMessage": "Scrivi il tuo messaggio...",
+  
+  "@_comment_dispute_list": "Testo Lista Controversie",
+  "disputesWillAppear": "Le controversie appariranno qui quando le apri dai tuoi scambi",
+  "failedLoadDisputes": "Impossibile caricare le controversie",
+  "retry": "Riprova",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)